packages/@atjson/source-gdocs-paste/src/converter/dropcaps.ts

```diff
@@ -0,0 +1,39 @@
+import Document from "@atjson/document";
+import { FontSize } from "../annotations/font-size";
+
+export function convertDropCaps(doc: Document) {
+  const paragraphs = doc.where({ type: "-offset-paragraph" });
+  const fontSizes = doc.where({ type: "-gdocs-ts_fs" });
+
+  paragraphs
+    .as("paragraph")
+    .join(
+      fontSizes.as("firstLetterSize"),
+      (paragraph, fontSize) =>
+        // font size annotations that only span the first letter of the paragraph
+        paragraph.start === fontSize.start &&
+        fontSize.end === fontSize.start + 1
+    )
+    .join(
+      fontSizes.as("adjacentSize"),
+      (
+        { firstLetterSize }: { firstLetterSize: FontSize[] },
```

Comment:
The type errors above would be resolved by https://github.com/CondeNast/atjson/pull/1837?

Reply:
yup!